### ISSUE TITLE
As a user I want to mount my S3 bucket and get/send files to it from my workspace

### ISSUE THREAD

**Issue by sanjayankur31** [OpenSourceBrain/OSBv2#536]
As in title

**Comment by filippomc** [comment]
I can think at different implementations of this. 
- public urls: already possible to paste a public url coming from a s3 bucket in the "add resource" form
- use the [s3 browser extension](https://www.npmjs.com/package/jupyterlab-s3-browser) to import files directly in jupyterlab 
- repository: that's a kind of stretched interpretation of the concept of repository but may work

**Comment by sanjayankur31** [comment]
The use case also includes private data. Let's say a lab has done their experiments and placed all their data in a lab specific s3 bucket. What is the easiest way to allow them to browse the bucket and import required files into their workspace?

Repositories would work, but if it's data that is still being analysed, they'll be private repositories---we don't have this concept yet.

**Comment by filippomc** [comment]
Yes in that case the simplest solution would be to enable the jupyter plugins. Adding dedicated importers would be probably better than repositories if sharing data with other labs is not a use case.

A dedicated importer could be initiated by a button like "From cloud storage" or similar here, where s3, dropbox, drive and potentially others can be selected.
![image](https://user-images.githubusercontent.com/46561561/177277550-3100576b-7689-4fa4-89ac-992305f110bc.png)

**Comment by filippomc** [comment]
Another option to investigate is to virtually mount the drives in the pod so it can be browsed as any other file
